codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/CodegenDelegator.kt

```diff
@@ -153,12 +168,12 @@ open class RustCrate(
  * Allowlist of modules that will be exposed publicly in generated crates
  */
 val DefaultPublicModules = setOf(
-    RustModule.public("error", documentation = "All error types that operations can return."),
-    RustModule.public("operation", documentation = "All operations that this crate can perform."),
+    RustModule.Error,
+    RustModule.Operation,
     RustModule.public("model", documentation = "Data structures used by operation inputs/outputs."),
     RustModule.public("input", documentation = "Input structures for operations."),
     RustModule.public("output", documentation = "Output structures for operations."),
```

Comment:
How come these three aren't defined as a `val` like `Error`, `Operation`, and `Config`?

Reply:
I only centralized the ones that are referenced in multiple places.